### DISCUSSION TITLE
Would you add GammaM0, GammaM1 and GammaM3 into class ConnectionSetup?

### DISCUSSION THREAD

**Discussion by rewiteq** [idea-statica/ideastatica-public#234] [Help Q&A] [answered]
Hi,
I want new parameters in  ConnectionSetup class. I need GammaM0, GammaM1 and GammaM3. It is possible to get value from this parameters?

Best regards
Witek

**Comment by nathanlukeIDEA** [comment]
Hi Witek,

Just to be sure on my end, you are trying to set these parameters in IOM so they are set correctly when you create a .ideaCon from IOM?

Cheers
Nathan

**Comment by rewiteq** [comment]
Hi Nathan,
Sorry γM3 exists inside class ConnectionSetup, but we need to set γM0, γM1 and γM2.

`Assembly IdeaRS.OpenModel, Version=21.0.0.0`

![gamma](https://user-images.githubusercontent.com/115782097/211353456-b54e7da2-9967-4f1f-b9ee-7569ca95edfe.png)
![iom](https://user-images.githubusercontent.com/115782097/211353520-19032f05-c170-49d0-9b0a-ee35e4d5ff23.png)

Best regards
Witek

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi Witek, 

This is actually already possible. SteelSetup is an abstract class you can use to set these values. Create a new SteelSetupECEN() class and assign it to the SteelSetup property. There is an example below:

```csharp        
public static void SetECENSetupValues(ConnectionSetup setup)
{
    setup.SteelSetup = new SteelSetupECEN()
    {
        GammaM0 = 0,
        GammaM1 = 1,
        GammaM2 = 2,
        GammaMfi = 3,
        GammaMu = 4
    };
}
```
Please let me know if you have problems with this.

Thanks
Nathan
